controllers/admin/AdminCartsController.php

```diff
@@ -472,7 +472,7 @@ public function ajaxProcessUpdateQty()
             // Don't try to use a product if not instanciated before due to errors
             if (isset($product) && $product->id) {
                 if (($id_product_attribute = Tools::getValue('id_product_attribute')) != 0) {
-                    if (!Product::isAvailableWhenOutOfStock($product->out_of_stock) && !Attribute::checkAttributeQty((int) $id_product_attribute, (int) $qty)) {
+                    if (!Product::isAvailableWhenOutOfStock($product->out_of_stock) && !ProductAttribute::checkAttributeQty((int) $id_product_attribute, (int) $qty)) {
```

Comment:
is this modification linked to phpstan requirements ?

Reply:
I think it's an omission when @atomiix has migrated to PHP 8.0.